packages/flutter/lib/src/material/material.dart

```diff
@@ -423,13 +425,13 @@ abstract class InkFeature {
   /// To add an ink feature to a piece of Material, obtain the
   /// [MaterialInkController] via [Material.of] and call
   /// [MaterialInkController.addInkFeature].
-  InkFeature({
-    this.renderer,
+  InkFeature._({
```

Comment:
We can add that in a future patch, but it already wasn't possible because you couldn't get all the required objects to call this constructor.  I can make it public again if you want, but that's not going to help anything (other than making it hard to document).